src/utils/constants/wallets.ts

```diff
@@ -25,7 +25,7 @@ const PARITY_SIGNER_COMPANION = {
 };
 
 const POLKADOTJS_WALLET = {
-  title: 'Polkadot.js (for developers)',
```

Comment:
Test was failing because we're not escaping the parentheses, but using regex in the test. Having parentheses is low priority so getting rid of them to avoid further manipulating the string for the test. I'll add a ticket to the backlog for this: potentially could be an issue with other tests making use of `new RegExp`.